packages/backend/src/Node.ts

```diff
@@ -79,6 +80,7 @@ export default class Node {
     this.address = address;
     this.authority = authority;
     this.networkId = networkId;
+    this.networkState = "Not received yet";
```

Comment:
I think I'd prefer the type to be a `Maybe<Types.NetworkState>` here with default `null` (analog to other optional values, like `location`). If there were to be a default string value, then it should be a string value that can be parsed as JSON.